src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -221,6 +219,7 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     static UINT WM_PRIV_VDCHANGED; // Message to get back on to the UI thread when virtual desktop changes
     static UINT WM_PRIV_VDINIT; // Message to get back to the UI thread when FancyZones are initialized
     static UINT WM_PRIV_EDITOR; // Message to get back on to the UI thread when the editor exits
+    static UINT WM_PRIV_VDUPDATE; // Message to get back on the UI thread on virtual desktops update

```

Comment:
Move it 2 lines above just to have the UINT WM_PRIV_VD* events one after the other.

Reply:
Moved, and rearranged a bit (init goes first, switch second, update third).